packages/ui/src/components/organisms/DynamicForm/components/RSJVInputAdaters/DateInputAdater/DateInputAdater.tsx

```diff
@@ -36,9 +37,19 @@ export const DateInputAdater: RJSFInputAdapter<string | null> = ({
     [onChange],
   );
 
+  const datePickerParams = useMemo(() => {
+    const params: DatePickerParams = {
+      disableFuture: uiSchema?.disableFutureDate,
+      disablePast: uiSchema?.disablePastDate,
+    };
+
+    return params;
+  }, [uiSchema?.disableFutureDate, uiSchema?.disablePastDate]);
```

Comment:
Does `useMemo` work the same as `useCallback` when it comes to importance of a value passed as props? Otherwise `datePickerParams` has no computation and there seems to be no effect depending on it.

Reply:
Yes, useMemo works same, except it executes passed function when useCallback just memoize it.

`datePickerParams` - is computation, because its generation of new object with params which updates only when related values are changed. So in case if something else in uiSchema will be changed this object will be same.